Misc/NEWS.d/next/Library/2020-05-20-12-53-20.bpo-9216.ps7Yf1.rst

```diff
@@ -0,0 +1,3 @@
+func:`hashlib.new` passed ``usedforsecurity`` to OpenSSL EVP constructor
```

Comment:
A user reading this shouldn't need to be aware of the internal implementation details behind the Python hashlib "usedforsecurity" construct.  I'd just simplify this news entry:

```
func:`hashlib.new` now passes the ``usedforsecurity`` flag to the internal OpenSSL hash constructors.
```

and probably omit the final  "test_hashlib and test_smtplib handle strict security policy better." part entirely as we don't _need_ a NEWS entry to mention updates to our internal test suite.  (though it is harmless to do so)